internal/pkg/template/workload.go

```diff
@@ -504,11 +506,18 @@ func (cfg *ALBListener) Aliases() []string {
 
 // RulePaths returns a slice consisting of all the routing paths mentioned across multiple listener rules.
 func (cfg *ALBListener) RulePaths() []string {
-	var rulePaths []string
+	rulePaths := make(map[string]struct{})
```

Comment:
This change is a patch unrelated to the import of alb right 🤔 ? Is our custom resource not functioning well if it gets a list of rule paths that contains duplicated ones?

Reply:
reverting the change. it's intented